_pages/en/it-picture-medium-term.md

```diff
@@ -28,7 +28,27 @@ In the following sections, we're trying to take the same end state and create "v
 
 #### Automatic Enrollment and Delivery
 
-- Number of services that require users to submit, or print, PDFs should be reduced by half
+Looking forward, the information required to enroll a Canadian into programs often exists via trusted organizations or other government departments, allowing for the benefits the programs to be made available without the need for a manual process on the behalf of citizens (more details can be found in the [long term plan](https://sara-sabr.github.io/ITStrategy/it-picture-long-term.html)). In order to move in this direction, IITB must move away from manual processes and towards automation and information systems allowing for the secure exchange of information. This aligns well with both the OneGC initiative and the [Policy on Service](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27916) which reads
```

Comment:
GENERAL COMMENT ON STRUCTURE
I would remove any mention of IITB in the Citizen section. Citizens do not see IITB but rather ESDC (medium term, long term they'll see GC). Instead I would restructure each sections as per the IITB-ESDC one:

- List perspective of Citizens that could be used as some metrics (e.g. tell us once, reduced wait time, improved automation)
- List the enablers required, with hyperlinks to the other sections (e.g. IITB moving away from manual processes, IITB permitted to allocate resources to continuous improvement)

I am unsure about leaving policy statements. This is getting EA world and can over complicate and lengthen the document. If we still want to keep policy statements, use something similar to reference-style links like
"blah blah blah blah [1](inline-reference-link-to-bottom-of-page-text)"

and at that inline-reference-link-to-bottom-of-page-text we have the quote of the Policy with the hyperlink reference to said policy

https://docs.gitlab.com/ee/user/markdown.html#links

Reply:
The GitLab documentation page seems to describe how to generate a link with the URL stored elsewhere in the page (reference-style, e.g. [1] bottom of page), not how to generate a reference link (e.g. "1") that will bring us to the reference at the bottom. But the documentation was kinda confusing ;)

We'll need a solution to generate these reference links. We could use anchors (e.g. "#reference") and manually add an HTML anchors at the bottom, e.g. "<a name="reference">1 Text of reference</a>"

Reply:
The idea is to use references like in documetns. E.g.
"In order to move in this direction, IITB must move away from manual processes and towards automation and information systems [1][2]"

[1] link to OneGC initiative
[2] link to Policy on Service and Digital